### ISSUE TITLE
Implement Create Category

### ISSUE THREAD

**Issue by BrandiATMuhkuh** [tagaprice/tagaprice#9]

**Comment by reihu** [comment]
I suppose you're talking about the GUI side because the DAO is already capable of creating Category stuff

**Comment by BrandiATMuhkuh** [comment]
Yes I do. I will create Tags for the issues to make clear which issues are for UI or DAO

**Comment by BrandiATMuhkuh** [comment]
Sorry. It is called Label and not Tag in the GitHub issue tracker
